bot/exts/events/advent_of_code/_helpers.py

```diff
@@ -368,11 +368,13 @@ def get_summary_embed(leaderboard: dict) -> discord.Embed:
     """Get an embed with the current summary stats of the leaderboard."""
     leaderboard_url = leaderboard["full_leaderboard_url"]
     refresh_minutes = AdventOfCode.leaderboard_cache_expiry_seconds // 60
+    refreshed_unix = int(datetime.datetime.fromisoformat(leaderboard["leaderboard_fetched_at"]).timestamp())
 
-    aoc_embed = discord.Embed(
-        colour=Colours.soft_green,
-        timestamp=datetime.datetime.fromisoformat(leaderboard["leaderboard_fetched_at"]),
-        description=f"*The leaderboard is refreshed every {refresh_minutes} minutes.*"
+    aoc_embed = discord.Embed(colour=Colours.soft_green)
+
+    aoc_embed.description = (
+        f"The leaderboard is refreshed every {refresh_minutes} minutes.\n"
+        f"Last Updated: <t:{refreshed_unix}:t>"
```

Comment:
So, there's actually a bug with the current implementation of how `leaderboard_fetched_at` is created, impacting this change.

If the bot doesn't run in UTC timezone, then the timestamp returned will be incorrect and X hours (where X is the utc offset) off from whatever timezone the bot is currently being run in. It's so far been unnoticed because in production it does run in UTC. However, this is still a bug as it's dependent on which timezone the bot runs in and impacts local development for those not in the same timezone as UTC.

The fix is changing line 335 in this file to:
```py
leaderboard_fetched_at = datetime.datetime.now(datetime.timezone.utc).isoformat()
```

Reply:
I've seen multiple places in both bots which rely on UTC timezone so I figured this was a non-issue.

Nonetheless, fixed in https://github.com/python-discord/sir-lancebot/pull/969/commits/4c451b170e0005726dac2489835045c1dc43ce62

Additionally did a rebase since there were changes to this cog.